pkg/odo/cli/cli.go

```diff
@@ -40,13 +40,16 @@ var (
  /  \__/    Find more information at https://odo.dev
  \__/`
 
-	odoExample = ktemplates.Examples(`  # Creating and deploying a Node.js project
-  git clone https://github.com/odo-devfiles/nodejs-ex && cd nodejs-ex
-  %[1]s create nodejs
-  %[1]s push
+	odoExample = ktemplates.Examples(`Initializing your component by taking your pick from multiple languages or frameworks:
```

Comment:
```suggestion
	odoExample = ktemplates.Examples(`Initialize your component by taking your pick from multiple languages or frameworks:
```

Reply:
Can you make the same changes to `rootDefaultHelp` on L85? This is provided when you simply run `odo`.